### PR TITLE
Add basic support for watchos

### PR DESCRIPTION
**Pull request by Ukalnins**
This still needs some testing on older macOS/iOS/Xcode versions, but created the PR to get some feedback.  

This PR should allow:
- Detecting watchOS devices connected trough iPhone
- Listing installed bundles
- Uninstalling/Installing on said watchOS device
- Starting debugger and application on said watchOS device

Looks like starting AFC service on watchOS is not possible, so copying files there and back again is also not possible.

### CODE REVIEW

**Review by gabebear — APPROVED** [review]
I think this looks good, leaving it up to @ivanhernandez13 to figure out what gets merged first. I think it makes sense to get this merged and follow with any needed fixes since it largely seems to work.

File: src/ios-deploy/ios-deploy.m
Comment (ivanhernandez13) [inline src/ios-deploy/ios-deploy.m:559]: Was there some confusion with this line?

**Comment by ivanhernandez13** [comment]
Thanks for the PR and sorry for the delay.

This change looks good to me as well, and works well from playing around with it using Xcode 12.4 + iOS 14.5 + watchOS 7.4.  @Ukalnins do you want to merge this in now or is there some additional changed you'd like to do?

I have a [very minor change](https://github.com/ios-control/ios-deploy/pull/503) I'd like to merge as well but can merge this in first if you'd like.

File: src/ios-deploy/ios-deploy.m
Comment (Ukalnins) [reply src/ios-deploy/ios-deploy.m:559]: Could not locate the directory initially, so got confused if it exists. Did verify it now, so I'll remove the comment.

**Comment by Ukalnins** [comment]
> This change looks good to me as well, and works well from playing around with it using Xcode 12.4 + iOS 14.5 + watchOS 7.4. @Ukalnins do you want to merge this in now or is there some additional changed you'd like to do?
> 
> I have a [very minor change](https://github.com/ios-control/ios-deploy/pull/503) I'd like to merge as well but can merge this in first if you'd like.

I would like to test that building on macOS-10.14 still works and iOS-10 also still works. Hadn't had the time unfortunately. I'll try to do it tomorrow and report back if I had any issues. 

You are welcome to merge your change and not wait for this.

File: src/ios-deploy/ios-deploy.m
Comment (Ukalnins) [reply src/ios-deploy/ios-deploy.m]: Removed the comment

**Comment by Ukalnins** [comment]
Looks like this PR does not break anything on Mojave nor iOS 10.x or tvOS, so I think this can be merged.

**Pull request merged**
